ios/FluentUI/Navigation/BadgeLabelButton.swift

```diff
@@ -208,5 +208,21 @@ class BadgeLabelButton: UIButton {
 
     @objc private func badgeValueDidChange() {
         updateBadgeLabel()
+        updateAccessibilityLabel()
+    }
+
+    private func updateAccessibilityLabel() {
+        guard let item = item else {
+            return
+        }
+        if let badgeValue = item.badgeValue, let itemAccessibilityLabel = item.accessibilityLabel {
```

Comment:
"let itemAccessibilityLabel = item.accessibilityLabel " i think this check is still incorrect. if there isn't an explict acessibilitylabel passed by the item, we still can use the button label itself to do the string concatenation

Reply:
I didn't get what do you mean by "button label itself". The button can also be image only button.

Also, I am following your suggestion about "change in behavior" because earlier also we just used to set accessibilityLabel = item.accessibilityLabel

Reply:
Done!!